pkg/user/handler.go

```diff
@@ -242,3 +245,80 @@ func (h Handler) FindById(c *gin.Context) {
 
 	c.JSON(http.StatusOK, userWithGroups)
 }
+
+// List user
+func (h Handler) List(c *gin.Context) {
+	// swagger:route GET /users listUsers
+	//
+	// Get users
+	//
+	// Get all users with the groups they belong to
+	//
+	// security:
+	//	oauth2:
+	//
+	// responses:
+	//	200: []User
+	//	401: Error
+	//	403: Error
+	//	404: Error
+	//	415: Error
+	users, err := h.userService.GetAll()
+	if err != nil {
+		_ = c.Error(err)
+		return
+	}
+
+	c.JSON(http.StatusOK, users)
+}
+
+// Delete user
+func (h Handler) Delete(c *gin.Context) {
+	// swagger:route DELETE /users/{id} deleteUser
+	//
+	// Delete user
+	//
+	// Delete user by id
+	//
+	// Security:
+	//	oauth2:
+	//
+	// Responses:
+	//	202:
+	//	401: Error
+	//	403: Error
+	//	404: Error
+	//	415: Error
+	idParam := c.Param("id")
+	id, err := strconv.ParseUint(idParam, 10, 32)
+	if err != nil {
+		badRequest := apperror.NewBadRequest("error parsing id")
+		_ = c.Error(badRequest)
+		return
+	}
+
+	user, err := handler.GetUserFromContext(c)
+	if err != nil {
+		return
```

Comment:
It seems like we always do `_ = c.Error(err)` on error when calling `GetUserFromContext`.

@tonsV2 should we change the signature to one like `GetPathParameter` so that we write

```go
	user, ok := handler.GetUserFromContext(c)
	if ok != nil {
       return
    }
```